[rospy.client][INFO] 2017-07-13 11:33:52,676: init_node, name[/nao_behaviours], pid[13315]
[xmlrpc][INFO] 2017-07-13 11:33:52,676: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2017-07-13 11:33:52,676: Started XML-RPC server [http://sandtray-hrilab:34408/]
[rospy.init][INFO] 2017-07-13 11:33:52,677: ROS Slave URI: [http://sandtray-hrilab:34408/]
[rospy.impl.masterslave][INFO] 2017-07-13 11:33:52,677: _ready: http://sandtray-hrilab:34408/
[rospy.registration][INFO] 2017-07-13 11:33:52,678: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2017-07-13 11:33:52,678: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2017-07-13 11:33:52,777: registered with master
[rospy.rosout][INFO] 2017-07-13 11:33:52,777: initializing /rosout core topic
[rospy.rosout][INFO] 2017-07-13 11:33:52,779: connected to core topic /rosout
[rospy.simtime][INFO] 2017-07-13 11:33:52,780: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2017-07-13 11:33:54,020: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2017-07-13 11:34:02,171: topic[/speech] adding connection to [/naoqi_driver_node], count 0
[rosout][WARNING] 2017-07-13 11:34:04,113: Waiting for frame /odom to be published...
[rospy.internal][INFO] 2017-07-13 11:34:04,138: topic[/tf] adding connection to [http://sandtray-hrilab:35171/], count 0
[rospy.internal][INFO] 2017-07-13 11:34:04,139: topic[/tf] adding connection to [http://sandtray-hrilab:39949/], count 1
[rospy.internal][INFO] 2017-07-13 11:34:04,141: topic[/tf] adding connection to [http://sandtray-hrilab:43402/], count 2
[rospy.internal][INFO] 2017-07-13 11:34:04,143: topic[/tf] adding connection to [http://sandtray-hrilab:43786/], count 3
[rospy.internal][INFO] 2017-07-13 11:34:04,146: topic[/tf] adding connection to [http://sandtray-hrilab:44434/], count 4
[rospy.internal][INFO] 2017-07-13 11:34:04,148: topic[/tf] adding connection to [http://sandtray-hrilab:39254/], count 5
[rospy.internal][INFO] 2017-07-13 11:34:04,148: topic[/tf] adding connection to [http://sandtray-hrilab:37806/], count 6
[rospy.internal][INFO] 2017-07-13 11:34:04,151: topic[/tf] adding connection to [http://192.168.2.103:44307/], count 7
[rosout][WARNING] 2017-07-13 11:34:04,614: No transform robot->sandtray. Robot not yet localised!
[rosout][WARNING] 2017-07-13 11:34:05,116: No transform robot->sandtray. Robot not yet localised!
[rosout][WARNING] 2017-07-13 11:34:05,617: No transform robot->sandtray. Robot not yet localised!
[rosout][WARNING] 2017-07-13 11:34:06,118: No transform robot->sandtray. Robot not yet localised!
[rosout][WARNING] 2017-07-13 11:34:06,620: No transform robot->sandtray. Robot not yet localised!
[rosout][WARNING] 2017-07-13 11:34:07,121: No transform robot->sandtray. Robot not yet localised!
[rosout][WARNING] 2017-07-13 11:34:07,622: No transform robot->sandtray. Robot not yet localised!
[rosout][WARNING] 2017-07-13 11:34:08,122: No transform robot->sandtray. Robot not yet localised!
[rosout][WARNING] 2017-07-13 11:34:08,623: No transform robot->sandtray. Robot not yet localised!
[rosout][WARNING] 2017-07-13 11:34:09,124: No transform robot->sandtray. Robot not yet localised!
[rosout][WARNING] 2017-07-13 11:34:09,625: No transform robot->sandtray. Robot not yet localised!
[rosout][WARNING] 2017-07-13 11:34:10,126: No transform robot->sandtray. Robot not yet localised!
[rosout][INFO] 2017-07-13 11:34:10,627: Ok! Starting robot behaviours.
[rospy.internal][INFO] 2017-07-13 11:34:10,655: topic[/poses] adding connection to [http://sandtray-hrilab:42979/], count 0
[rospy.internal][WARNING] 2017-07-13 12:09:11,723: Unknown error initiating TCP/IP socket to 192.168.2.103:35922 (http://192.168.2.103:44307/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 555, in connect
    self.socket.connect((dest_addr, dest_port))
  File "/usr/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
error: [Errno 111] Connection refused

[rospy.internal][INFO] 2017-07-13 12:09:11,723: topic[/tf] removing connection to http://192.168.2.103:44307/
[rospy.internal][WARNING] 2017-07-13 12:09:12,120: Unknown error initiating TCP/IP socket to sandtray-hrilab:38105 (http://sandtray-hrilab:44434/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 555, in connect
    self.socket.connect((dest_addr, dest_port))
  File "/usr/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
error: [Errno 111] Connection refused

[rospy.internal][INFO] 2017-07-13 12:09:12,121: topic[/tf] removing connection to http://sandtray-hrilab:44434/
[rospy.internal][INFO] 2017-07-13 12:09:12,175: topic[/speech] removing connection to /naoqi_driver_node
[rospy.core][INFO] 2017-07-13 12:09:12,741: signal_shutdown [signal-2]
[rosout][ERROR] 2017-07-13 12:09:12,743: bad callback: <function on_pose at 0x7fe948f66938>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/opt/ros/kinetic/lib/freeplay_sandbox/nao_behaviours", line 214, in on_pose
    point_at(pose)
  File "/opt/ros/kinetic/lib/freeplay_sandbox/nao_behaviours", line 142, in point_at
    motionProxy.setAngles(names, angles, fractionMaxSpeed)
  File "/home/hri/nao/pynaoqi-python2.7-2.1.4.13-linux64/naoqi.py", line 301, in __call__
    return self.__wrapped__.method_missing(self.__method__, *args, **kwargs)
  File "/home/hri/nao/pynaoqi-python2.7-2.1.4.13-linux64/naoqi.py", line 371, in method_missing
    raise e
RuntimeError: 	ALMotion::setAngles
	Socket disconnected

[rosout][ERROR] 2017-07-13 12:09:12,744: bad callback: <function on_pose at 0x7fe948f66938>
Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/opt/ros/kinetic/lib/freeplay_sandbox/nao_behaviours", line 214, in on_pose
    point_at(pose)
  File "/opt/ros/kinetic/lib/freeplay_sandbox/nao_behaviours", line 133, in point_at
    angles = arm_ik(targetpose)
  File "/opt/ros/kinetic/lib/freeplay_sandbox/nao_behaviours", line 98, in arm_ik
    pose = tl.transformPose(reference_frame, targetpose)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/tf/listener.py", line 184, in transformPose
    mat44 = self.asMatrix(target_frame, ps.header)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/tf/listener.py", line 71, in asMatrix
    translation,rotation = self.lookupTransform(target_frame, hdr.frame_id, hdr.stamp)
ExtrapolationException: Lookup would require extrapolation into the past.  Requested time 1499944117.724346638 but the earliest data is at time 1499944141.981090307, when looking up transform from frame [sandtray] to frame [RShoulder]

[rospy.internal][INFO] 2017-07-13 12:09:12,747: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2017-07-13 12:09:12,747: topic[/poses] removing connection to http://sandtray-hrilab:42979/
[rospy.topics][ERROR] 2017-07-13 12:09:12,748: Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 838, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.internal][WARNING] 2017-07-13 12:09:12,748: Unknown error initiating TCP/IP socket to sandtray-hrilab:44580 (http://sandtray-hrilab:42979/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.internal][INFO] 2017-07-13 12:09:12,748: topic[/tf] removing connection to http://sandtray-hrilab:35171/
[rospy.internal][INFO] 2017-07-13 12:09:12,749: topic[/tf] removing connection to http://sandtray-hrilab:39949/
[rospy.internal][INFO] 2017-07-13 12:09:12,749: topic[/tf] removing connection to http://sandtray-hrilab:43402/
[rospy.internal][INFO] 2017-07-13 12:09:12,749: topic[/tf] removing connection to http://sandtray-hrilab:43786/
[rospy.internal][INFO] 2017-07-13 12:09:12,749: topic[/tf] removing connection to http://sandtray-hrilab:39254/
[rospy.internal][INFO] 2017-07-13 12:09:12,749: topic[/tf] removing connection to http://sandtray-hrilab:37806/
[rospy.impl.masterslave][INFO] 2017-07-13 12:09:12,750: signal-2
[rospy.internal][ERROR] 2017-07-13 12:09:12,750: Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 729, in receive_once
    self.stat_bytes += recv_buff(sock, b, p.buff_size)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 103, in recv_buff
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.core][INFO] 2017-07-13 12:09:13,365: signal_shutdown [atexit]
